.github/workflows/build.yml

```diff
@@ -1,9 +1,6 @@
-on:
-  push:
-    branches:
-    - master
-    tags-ignore:
-    - v*
+on: push #:
+#    tags-ignore:
+#    - v*
```

Comment:
The `tags-ignore` part should be left in, since we have a separate github action for releases tagged with `v*`.